tasks/main.yml

```diff
@@ -113,8 +113,36 @@
     name: "{{ camac_releasedir }}/camac/django/"
     extra_args: "--upgrade"
 
+- name: reset django_migrations table
+  shell: |
+    psql \
+        -h {{ camac_db_host }} \
+        -p {{ camac_db_port }} \
+        -U {{ camac_db_username }} \
+        -d {{ camac_db_dbname }} \
+        -c "TRUNCATE django_migrations"
+  when: camac_db_run_fake_migrations and camac_db_fake_migrations is defined
+  environment:
+    PGPASSWORD: "{{ camac_db_password }}"
+
+- name: run camac fake migrations
+  shell: "python3 {{ camac_releasedir }}/camac/django/manage.py migrate --fake {{ item.app }} {{ item.migration }}"
+  when: camac_db_run_fake_migrations and camac_db_fake_migrations is defined
+  environment:
+    DJANGO_SETTINGS_MODULE: "camac.settings"
+    DJANGO_DATABASE_ENGINE: "django.db.backends.postgresql_psycopg2"
```

Comment:
/cc @sliverc I think we talked about dropping this once ...

Reply:
yep this configuration doesn't exist any more so can also be dropped in ansible

Reply:
I've removed `DJANGO_DATABASE_ENGINE`.